.github/workflows/crowdin-download.yml

```diff
@@ -0,0 +1,136 @@
+name: Crowdin Download
+on:
+  workflow_dispatch:
+  # schedule:
+  #   # runs every day at 12:00 noon UTC
+  #   - cron: '0 12 * * *'
+
+  jobs:
+    i18n-download:
+      name: Download
+      runs-on: ubuntu-20.04
+
+      steps:
+        - name: Checkout Source Files
+          uses: actions/checkout@v2
+          with:
+            token: ${{ secrets.CROWDIN_CAMPERBOT_PAT }}
+
+        - name: Generate Crowdin Config
+          uses: freecodecamp/crowdin-action@main
+          env:
+            CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_SERVICE_TOKEN }}
+            CROWDIN_API_URL: "https://freecodecamp.crowdin.com/api/v2/"
+            CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID_RIR }}
```

Comment:
Now that I have changed this, where should it go?

Reply:
I loaded both of these into the repo secrets